src/managed/OpenLiveWriter.BlogClient/Providers/BlogProvidersB5.xml

```diff
@@ -584,6 +584,7 @@
 		]]>
     </postApiUrl>
     <options>
+      <supportsMultipleCategories>Yes</supportsMultipleCategories>
```

Comment:
ohh, its a mistake.

Anyways, SupportsPostAsDraft's defait value is True. That's why drafting is working now. I'll update this.